libclc/cmake/modules/AddLibclc.cmake

```diff
@@ -39,6 +39,10 @@ function(compile_to_bc)
     set( TARGET_ARG "-target" ${ARG_TRIPLE} )
   endif()
 
+  # Ensure the directory we are told to output to exists
+  get_filename_component( ARG_OUTPUT_DIR ${ARG_OUTPUT} DIRECTORY )
```

Comment:
I thought there was a way to express this as a DEPENDS of the command

Reply:
Yeah, there very well may be. I couldn't find anything obvious, but I'm sure I'll keep iterating on the build system over time so I'll keep an eye out.